float32 x
float32 y
float32 theta
float32 y_wall
bool wall
bool right_sensor